README.md

```diff
@@ -0,0 +1,256 @@
+```
+
+Then, all the others information related to your Git repository or debug level
+
+```ruby
+# Default value for :scm is :git
+set :scm, :git
+
+# Default value for :log_level is :debug
+set :log_level, :debug
+```
+
+Finally, set the deployment to use the proper Drupal 8 strategy
+
+```ruby
+namespace :deploy do
+  # Ensure everything is ready to deploy.
+  after "deploy:check:directories", "drupal:db:backup:check"
+
+  # Backup the database before starting a deployment and rollback on fail.
+  # before :starting, "drupal:db:backup"
+  # before :failed, "drupal:db:rollback"
+  # before :cleanup, "drupal:db:backup:cleanup"
+
+  # Set the maintenance Mode on your Drupal online project when deploying.
+  after :updated, "drupal:maintenance:on"
+
+  # Must updatedb before import configurations, E.g. when composer install new
+  # version of Drupal and need updatedb scheme before importing new config.
+  # This is executed without raise on error, because sometimes we need to do drush config-import before updatedb.
+  after :updated, "drupal:updatedb:silence"
+
+  # Remove the cache after the database update
+  after :updated, "drupal:cache:clear"
+  after :updated, "drupal:config:import"
+
+  # Update the database after configurations has been imported.
+  after :updated, "drupal:updatedb"
+
+  # Clear your Drupal 8 cache.
+  after :updated, "drupal:cache:clear"
+
+  # Disable the maintence on the Drupal project.
+  after :updated, "drupal:maintenance:off"
+
+  # Ensure permissions are properly set.
+  after :updated, "drupal:permissions:recommended"
+  after :updated, "drupal:permissions:writable_shared"
+
+
+  # Fix the release permissions (due to Drupal restrictive permissions)
+  # before deletting old release.
+  before :cleanup, :fix_permission do
+    on roles(:app) do
+      releases = capture(:ls, '-xtr', releases_path).split
+      if releases.count >= fetch(:keep_releases)
+        directories = (releases - releases.last(fetch(:keep_releases)))
+        if directories.any?
+          directories_str = directories.map do |release|
+            releases_path.join(release)
+          end.join(" ")
+          execute :chmod, '-R' ,'ug+w', directories_str
+        end
+      end
+    end
+  end
+end
+```
+
+You may now can configure your `staging.rb` and `production.rb` strategy, has you will certainly deploy on different environment
+
+```shell
+vi config/deploy/staging.rb
+```
+
+```ruby
+# staging.example.org
+set :deploy_to, '/home/example.org/www/staging.example.org'
+
+# set a branch for this release
+set :branch, 'dev'
+
+# Map composer and drush commands
+# NOTE: If stage have different deploy_to
+# you have to copy those line for each <stage_name>.rb
+# See https://github.com/capistrano/composer/issues/22
+SSHKit.config.command_map[:composer] = shared_path.join("composer.phar")
+SSHKit.config.command_map[:drush] = shared_path.join("vendor/bin/drush")
```

Comment:
```suggestion
SSHKit.config.command_map[:composer] = -> { shared_path.join('composer.phar') }
SSHKit.config.command_map[:drush] = -> { release_path.join('vendor/bin/drush') }
```

Use lambda to build the path at runtime and use drush from project vendors. This change should allow us to move this in the `deploy.rb` also (testing in progress).